app/scripts/controllers/tasks.js

```diff
@@ -4,12 +4,20 @@
 /* global valueNotEmpty:false */
 
 angular.module('openhimConsoleApp')
-  .controller('TasksCtrl', function ($scope, $modal, $location, Api, Alerting, $route) {
+  .controller('TasksCtrl', function ($scope, $modal, $location, $interval, Api, Alerting, $route) {
 
     /**********************************************/
     /**         Initial load functions           **/
     /**********************************************/
 
+
+
+    var tasksInterval = $interval(function() {
+      $scope.refreshTasksList();
+    }, 5000);
```

Comment:
This seems to make the task list 'flash' every 5s. If we could make this happen in the background and then just load in the new data it would be ideal. Else, maybe we remove this until we can figure that out.